bubble-wrap-style.yaml

```diff
@@ -3194,27 +3362,79 @@ layers:
-            draw:
+            filter: 
+                global.ux_point_of_view_kind_filter_disputed_etc
+                #- kind: [disputed, lease_limit, line_of_control, overlay_limit]
+                #- kind: [disputed_breakaway, disputed_claim, disputed_elusive, disputed_reference_line]
+            draw:
+                #lines: 
+#                    order: global.feature_order
+#                    visible: true
+#                    color: blue
+#                    width: 2px
                 dashed:
+                    visible: true
                     order: global.feature_order
                     color: global.country_boundary
-                    width: [[1, 0.2px], [2, 0.5px], [4, 1.0px], [9, 1.5px], [14, 2.5px], [16, 3.5px], [17, 10m]]
+                    width: [[1, 0.2px], [2, 0.5px], [5, 1.5px], [6, 2.0px], [8, 2.5px], [16, 3.5px], [17, 10m]]
             disputed:
-                filter: { kind: [disputed, line_of_control] }
-                draw:
+                filter: 
+                    #- kind: [disputed, line_of_control]
+                    #- kind: [disputed_breakaway, disputed_claim, disputed_elusive, disputed_reference_line]
+                    global.ux_point_of_view_kind_filter_disputed_star
+                draw:
+                    #lines: 
+#                        order: global.feature_order
+#                        visible: true
+#                        color: red
+#                        width: 2px
                     dashed:
                         visible: false
                     dashed_big:
+                        visible: true
                         order: global.feature_order
                         color: global.country_boundary
                         #width: [[1, 0.2px], [2, 0.5px], [4, 1.0px], [9, 1.5px], [14, 2.5px], [16, 3.5px], [17, 10m]]
                         width: [[1, 0.5px], [2, 1.0px], [4, 1.5px], [9, 2.5px], [14, 3.5px], [16, 4.5px], [17, 14m]]
+                        #width: 20px
+#            disputed_star_early:
+#                filter: 
+#                    #kind: [disputed_breakaway, disputed_claim, disputed_elusive, disputed_reference_line]
+#                    global.ux_point_of_view_kind_filter_disputed_star
+#                    $zoom: [5,6]
+#                draw:
+#                    dashed:
+#                        width: [[4, 0px], [5, 1.0px], [8, 2.5px], [16, 3.5px], [17, 10m]]
+#                    dashed_big:
+#                        width: [[5, 0px], [5, 1.0px], [9, 2.5px], [14, 3.5px], [16, 4.5px], [17, 14m]]
+
+        unrecognized:
+            filter: 
+                - global.ux_point_of_view_kind_filter_unrecognized
+            draw:
+                 lines:
+                     visible: false
+                 text-blend-order:
+                     visible: false
+                 dashed:
+                     visible: false
+                 dashed_big:
+                     visible: false
+
+        map_unit:
```

Comment:
Added a new kind (like for England, Wales)

Reply:
huh, cool